config/dotenv.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(/* env */) {
+    return {
+      clientAllowedKeys: [],
```

Comment:
From the `ember-cli-dotenv` documentation, I wasn't quite sure what `clientAllowedKeys` does. I was able to run the app and fetch data without specifying `'GITHUB_API_KEY'` in `clientAllowedKeys`.

Do you know its purpose, by any chance?

Reply:
No idea, In any app I have built it has always remained empty. There is not a lot of documentation on this 🤔